pt-BR/mail/chrome/messenger/am-offline.dtd

```diff
@@ -1,10 +1,10 @@
 <!ENTITY doNotDownloadPop3Movemail.label "Para economizar espaço em disco, não receber:">
-<!ENTITY offlineNewFolder.label "When I create new folders, select them for offline use">
+<!ENTITY offlineNewFolder.label "Quando eu criar novas pastas, habilitar o uso offline nelas">
```

Comment:
I agree, with the exception of "as mesmas", and the sentence is weird with habilitar used in this way. I'll change to: "Quando eu criar novas pastas, disponibilizá-las para uso em modo offline". Please let me know if you agree.